TokenProject/contracts/Token.vy

```diff
@@ -0,0 +1,183 @@
+# @version 0.3.3
+
+from vyper.interfaces import ERC20
+
+implements: ERC20
+
+# ERC20 Token Metadata
+NAME: constant(String[20]) = "TokenTest"
+SYMBOL: constant(String[5]) = "TKN"
+DECIMALS: constant(uint8) = 18
+
+# ERC20 State Variables
+totalSupply: public(uint256)
+balanceOf: public(HashMap[address, uint256])
+allowance: public(HashMap[address, HashMap[address, uint256]])
+
```

Comment:
@unparalleled-js I think you added some stuff you didn't intend to be in the template?

Reply:
fixed! It was related to an issue of running the tests from outside the tests directory so I made a fixture that set the currdir there as well so they won't show up again.